src/eosjs2-api.ts

```diff
@@ -93,7 +93,7 @@ export class Api {
         }));
     }
 
-    async pushTransaction({ blocksBehind, expireSeconds, actions, ...transaction }: any) {
+    async fillTaposFields({ blocksBehind, expireSeconds, actions, ...transaction }: any) {
```

Comment:
Had a conversation with todd this morning and we should change this signature and our examples to something more like

{transactionstuff}

async addCalculatedTaposFields(transaction, {blocksBehind, expireSeconds})

Since blockBehind and expireSeconds aren't actual tapos fields it makes it confusing when we have them as properties on the transaction object we're dealing with.

Reply:
I agree; it is confusing. Either we force the devs to call this separately (more boilerplate), or maybe there's a way to pass a second config arg to `signAndPushTransaction`. I was a fan of how the first `eosjs` allowed you to specify `broadcast: false`. Fewer separate methods to remember: signTx, pushTx, signAndPushTx, signAndPushButDon't...jk. Of course, the original name would no longer be `signAndPushTransaction`.